docker-compose/docker-compose-dev.yml

```diff
@@ -0,0 +1,10 @@
+version: '3'
+services:
+  todo-app-development:
+    image: <image>:<tag>
```

Comment:
Instead of doing a docker build command and then referencing the image here, you can put this here:
```
    build: 
      context: .
      target: development
```

And then build & run it with one command: `docker-compose -f docker-compose-dev.yml --build up`